lib/ansible/modules/windows/win_find.ps1

```diff
@@ -17,7 +17,7 @@ $age_stamp = Get-AnsibleParam -obj $params -name 'age_stamp' -default 'mtime' -V
 $file_type = Get-AnsibleParam -obj $params -name 'file_type' -default 'file' -ValidateSet 'file','directory'
 $follow = Get-AnsibleParam -obj $params -name 'follow' -type "bool" -default $false
 $hidden = Get-AnsibleParam -obj $params -name 'hidden' -type "bool" -default $false
-$patterns = Get-AnsibleParam -obj $params -name 'patterns'
+$patterns = Get-AnsibleParam -obj $params -name 'patterns' -aliases "regex","regexp"
```

Comment:
I'm not 100% sure whether this should be `regex` or `regexp`. Technially a pattern can be a file glob like pattern (default) and the `use_regex` option changes that to a regex pattern.

Reply:
Yeah that is why I added them as aliases - it might be readable for someone  using the regex to call the pattern regex